src/test/resources/regressions/features/int_sizes/int-assign-fail1.gobra

```diff
@@ -0,0 +1,24 @@
+package ints
```

Comment:
Add licenses to all new files.

Reply:
I added licenses to all new test files in commit 8cb139e. I did not add a license to the Bounds file because I'm planning to remove it in another commit before the PR gets merged.